docs/core/tools/dotnet-build.md

```diff
@@ -69,21 +69,21 @@ In addition to its options, the `dotnet build` command accepts MSBuild options,
 
 Running `dotnet build` is equivalent to running `dotnet msbuild -restore`; however, the default verbosity of the output is different.
 
-[!INCLUDE [cli-advertising-manifests](../../../includes/cli-advertising-manifests.md)]
+- [!INCLUDE [cli-advertising-manifests](../../../includes/cli-advertising-manifests.md)]
```

Comment:
I don't think this is supposed to be a bullet since the content is a ### header

```suggestion
[!INCLUDE [cli-advertising-manifests](../../../includes/cli-advertising-manifests.md)]
```